### PR TITLE
[Mellanox] SN4410 support

### PR DESCRIPTION
**Pull request by vadymhlushko-mlnx**
Signed-off-by: Vadym Hlushko <vadymh@nvidia.com>

**- Why I did it**

Add support for Mellanox Spectrum-3 based 100GbE/400GbE 1U. 24 QSFP-DD28 and 8 QSFP-DD ports

**- How I did it**

Added all config files that are needed to support SN4410.

**- How to verify it**

Need to check:
- if there are no errors in logs via serial connection
- if there are no errors in logs via ssh connection
- interfaces creation
- basic switching
- basic routing 

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201911
- [x] master

**- Description for the changelog**

Added files like for other platforms (e.g. SN4700) under - **sonic-buildimage/device/mellanox/x86_64-mlnx_msn4410-r0** directory.

### CODE REVIEW

**Comment by vadymhlushko-mlnx** [comment]
retest this please

**Comment by volodymyrsamotiy** [comment]
Should be merged after Azure/sonic-buildimage#5767 is merged to master

**Comment by keboliu** [comment]
retest this please

**Comment by lguohan** [comment]
retest baseimage please

**Comment by liat-grozovik** [comment]
@vadymhlushko-mlnx  please handle LGTM issues

**Comment by liat-grozovik** [comment]
retest mellanox please

**Review by liat-grozovik — APPROVED** [review]

**Comment by lguohan** [comment]
any missing data we need for a platform?

**Comment by liat-grozovik** [comment]
> any missing data we need for a platform?

all should be there but I didnt merge as sonic-mgmt is not fully updated. so waiting for this PR to be available first and then we can merge both.

**Review by liat-grozovik — CHANGES_REQUESTED** [review]
missing pcied file

**Comment by vadymhlushko-mlnx** [comment]
retest vsimage please

**Comment by vadymhlushko-mlnx** [comment]
retest vsimage please

File: platform/mellanox/mlnx-platform-api/sonic_platform/thermal.py
Comment (liat-grozovik) [inline platform/mellanox/mlnx-platform-api/sonic_platform/thermal.py:272]: @vadymhlushko-mlnx why this one was added in the middle and not on the end?
BTW, this list is only for non-default systems? i dont see sn4700 and i wonder if this should be added.

**Comment by liat-grozovik** [comment]
@vadymhlushko-mlnx please provide the reference to the sonic-mgmt with the sensor test.

File: platform/mellanox/mlnx-platform-api/sonic_platform/thermal.py
Comment (vadymhlushko-mlnx) [reply platform/mellanox/mlnx-platform-api/sonic_platform/thermal.py:272]: I will commit this change to the end.

SN4700 was already added - [thermal.py](https://github.com/Azure/sonic-buildimage/blob/master/platform/mellanox/mlnx-platform-api/sonic_platform/thermal.py) please check

Comment (vadymhlushko-mlnx) [reply platform/mellanox/mlnx-platform-api/sonic_platform/thermal.py]: Done

**Comment by vadymhlushko-mlnx** [comment]
> @vadymhlushko-mlnx please provide the reference to the sonic-mgmt with the sensor test.

@liat-grozovik could you please clarify - what should I provide?

**Review by liat-grozovik — APPROVED** [review]

**Comment by liat-grozovik** [comment]
@jleveque any further comment? let me know if i missed a file :-)

**Review by jleveque — APPROVED** [review]
LGTM

**Pull request merged**
